djangoProject/djangoProject/settings.py

```diff
@@ -75,6 +75,34 @@
 
 WSGI_APPLICATION = 'djangoProject.wsgi.application'
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s %(levelname)s %(name)s %(message)s'
+        }
+    },
+    'handlers': {
+        'console': {
+            'level': 'INFO',
+            'class': 'logging.StreamHandler',
+            'formatter': 'standard',
+            'filters': []
+        }
+    },
+    'loggers': {
+        logger_name : {
+            'level': 'WARNING',
+            'propagate': True,
+        } for logger_name in ('django', 'django.request', 'django.db.backends', 'django.template', 'communications', 'dashboard', 'djangoProject', 'profiles', 'projects', 'users')
+    },
+    'root': {
+        'level': 'DEBUG',
+        'handlers': ['console'],
+    }
```

Comment:
For production, consider raising the root logger's level to `INFO` to reduce verbosity